ERROR - 2024-11-29 17:32:04 --> mysqli_sql_exception: Can't create table `808_starter`.`clients` (errno: 150 "Foreign key constraint is incorrectly formed") in X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `c...', 0)
#1 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 X:\00.templates\ci4-starter\dist\app\Database\Migrations\2024-11-27-063842_Cms.php(132): CodeIgniter\Database\Forge->createTable('clients')
#5 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(867): App\Database\Migrations\Cms->up()
#6 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(66): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(52): App\Commands\Install->runMigrations()
#10 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Commands.php(70): App\Commands\Install->run(Array)
#11 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', Array)
#12 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#13 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 X:\00.templates\ci4-starter\dist\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2024-11-29 17:32:07 --> mysqli_sql_exception: Table '808_starter.leadsources' doesn't exist in X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `le...', 0)
#1 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `le...')
#2 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `le...')
#3 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `le...', Array, false)
#4 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 X:\00.templates\ci4-starter\dist\app\Database\Seeds\LeadSeeder.php(56): CodeIgniter\BaseModel->save(Array)
#9 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\LeadSeeder->run()
#10 X:\00.templates\ci4-starter\dist\app\Database\Seeds\A2cms.php(12): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#11 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
#12 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#13 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(53): App\Commands\Install->runSeeder()
#14 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Commands.php(70): App\Commands\Install->run(Array)
#15 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', Array)
#16 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#17 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#18 X:\00.templates\ci4-starter\dist\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#19 {main}
CRITICAL - 2024-11-29 17:32:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Table '808_starter.leadsources' doesn't exist
[Method: CLI, Route: starter:install]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (:title:, :sort:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Database\Seeds\LeadSeeder.php(56): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\LeadSeeder->run()
 8 APPPATH\Database\Seeds\A2cms.php(12): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\LeadSeeder')
 9 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
10 APPPATH\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\A2cms')
11 APPPATH\Commands\Install.php(53): App\Commands\Install->runSeeder()
12 SYSTEMPATH\CLI\Commands.php(70): App\Commands\Install->run([])
13 SYSTEMPATH\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', [])
14 SYSTEMPATH\Boot.php(351): CodeIgniter\CLI\Console->run()
15 SYSTEMPATH\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
16 ROOTPATH\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
CRITICAL - 2024-11-29 17:32:07 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table '808_starter.leadsources' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (\'Google\', 1, \'2024-11-29 17:32:07\', \'2024-11-29 17:32:07\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (\'Google\', 1, \'2024-11-29 17:32:07\', \'2024-11-29 17:32:07\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (:title:, :sort:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Database\Seeds\LeadSeeder.php(56): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\LeadSeeder->run()
10 APPPATH\Database\Seeds\A2cms.php(12): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\LeadSeeder')
11 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
12 APPPATH\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\A2cms')
13 APPPATH\Commands\Install.php(53): App\Commands\Install->runSeeder()
14 SYSTEMPATH\CLI\Commands.php(70): App\Commands\Install->run([])
15 SYSTEMPATH\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', [])
16 SYSTEMPATH\Boot.php(351): CodeIgniter\CLI\Console->run()
17 SYSTEMPATH\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
18 ROOTPATH\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
CRITICAL - 2024-11-29 17:32:07 --> [Caused by] mysqli_sql_exception: Table '808_starter.leadsources' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (\'Google\', 1, \'2024-11-29 17:32:07\', \'2024-11-29 17:32:07\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (\'Google\', 1, \'2024-11-29 17:32:07\', \'2024-11-29 17:32:07\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (\'Google\', 1, \'2024-11-29 17:32:07\', \'2024-11-29 17:32:07\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (:title:, :sort:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Database\Seeds\LeadSeeder.php(56): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\LeadSeeder->run()
11 APPPATH\Database\Seeds\A2cms.php(12): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\LeadSeeder')
12 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
13 APPPATH\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\A2cms')
14 APPPATH\Commands\Install.php(53): App\Commands\Install->runSeeder()
15 SYSTEMPATH\CLI\Commands.php(70): App\Commands\Install->run([])
16 SYSTEMPATH\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', [])
17 SYSTEMPATH\Boot.php(351): CodeIgniter\CLI\Console->run()
18 SYSTEMPATH\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
19 ROOTPATH\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
ERROR - 2024-11-29 17:36:08 --> mysqli_sql_exception: Table 'client_groups' already exists in X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `c...', 0)
#1 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 X:\00.templates\ci4-starter\dist\app\Database\Migrations\2024-11-27-063842_Cms.php(93): CodeIgniter\Database\Forge->createTable('client_groups')
#5 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(867): App\Database\Migrations\Cms->up()
#6 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(66): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(52): App\Commands\Install->runMigrations()
#10 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Commands.php(70): App\Commands\Install->run(Array)
#11 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', Array)
#12 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#13 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 X:\00.templates\ci4-starter\dist\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2024-11-29 17:36:11 --> mysqli_sql_exception: Table '808_starter.leadsources' doesn't exist in X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `le...', 0)
#1 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `le...')
#2 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `le...')
#3 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `le...', Array, false)
#4 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 X:\00.templates\ci4-starter\dist\app\Database\Seeds\LeadSeeder.php(56): CodeIgniter\BaseModel->save(Array)
#9 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\LeadSeeder->run()
#10 X:\00.templates\ci4-starter\dist\app\Database\Seeds\A2cms.php(12): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#11 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
#12 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#13 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(53): App\Commands\Install->runSeeder()
#14 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Commands.php(70): App\Commands\Install->run(Array)
#15 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', Array)
#16 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#17 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#18 X:\00.templates\ci4-starter\dist\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#19 {main}
CRITICAL - 2024-11-29 17:36:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Table '808_starter.leadsources' doesn't exist
[Method: CLI, Route: starter:install]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (:title:, :sort:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Database\Seeds\LeadSeeder.php(56): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\LeadSeeder->run()
 8 APPPATH\Database\Seeds\A2cms.php(12): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\LeadSeeder')
 9 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
10 APPPATH\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\A2cms')
11 APPPATH\Commands\Install.php(53): App\Commands\Install->runSeeder()
12 SYSTEMPATH\CLI\Commands.php(70): App\Commands\Install->run([])
13 SYSTEMPATH\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', [])
14 SYSTEMPATH\Boot.php(351): CodeIgniter\CLI\Console->run()
15 SYSTEMPATH\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
16 ROOTPATH\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
CRITICAL - 2024-11-29 17:36:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table '808_starter.leadsources' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (\'Google\', 1, \'2024-11-29 17:36:11\', \'2024-11-29 17:36:11\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (\'Google\', 1, \'2024-11-29 17:36:11\', \'2024-11-29 17:36:11\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (:title:, :sort:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Database\Seeds\LeadSeeder.php(56): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\LeadSeeder->run()
10 APPPATH\Database\Seeds\A2cms.php(12): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\LeadSeeder')
11 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
12 APPPATH\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\A2cms')
13 APPPATH\Commands\Install.php(53): App\Commands\Install->runSeeder()
14 SYSTEMPATH\CLI\Commands.php(70): App\Commands\Install->run([])
15 SYSTEMPATH\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', [])
16 SYSTEMPATH\Boot.php(351): CodeIgniter\CLI\Console->run()
17 SYSTEMPATH\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
18 ROOTPATH\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
CRITICAL - 2024-11-29 17:36:11 --> [Caused by] mysqli_sql_exception: Table '808_starter.leadsources' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (\'Google\', 1, \'2024-11-29 17:36:11\', \'2024-11-29 17:36:11\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (\'Google\', 1, \'2024-11-29 17:36:11\', \'2024-11-29 17:36:11\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (\'Google\', 1, \'2024-11-29 17:36:11\', \'2024-11-29 17:36:11\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `leadsources` (`title`, `sort`, `created_at`, `updated_at`) VALUES (:title:, :sort:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Database\Seeds\LeadSeeder.php(56): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\LeadSeeder->run()
11 APPPATH\Database\Seeds\A2cms.php(12): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\LeadSeeder')
12 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
13 APPPATH\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\A2cms')
14 APPPATH\Commands\Install.php(53): App\Commands\Install->runSeeder()
15 SYSTEMPATH\CLI\Commands.php(70): App\Commands\Install->run([])
16 SYSTEMPATH\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', [])
17 SYSTEMPATH\Boot.php(351): CodeIgniter\CLI\Console->run()
18 SYSTEMPATH\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
19 ROOTPATH\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
ERROR - 2024-11-29 17:38:36 --> mysqli_sql_exception: Table 'client_groups' already exists in X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `c...', 0)
#1 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 X:\00.templates\ci4-starter\dist\app\Database\Migrations\2024-11-27-063842_Cms.php(93): CodeIgniter\Database\Forge->createTable('client_groups')
#5 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(867): App\Database\Migrations\Cms->up()
#6 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(66): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(52): App\Commands\Install->runMigrations()
#10 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Commands.php(70): App\Commands\Install->run(Array)
#11 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', Array)
#12 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#13 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 X:\00.templates\ci4-starter\dist\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2024-11-29 17:38:39 --> mysqli_sql_exception: Table '808_starter.clients_contacts' doesn't exist in X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `cl...', 0)
#1 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cl...')
#2 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cl...')
#3 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cl...', Array, false)
#4 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 X:\00.templates\ci4-starter\dist\app\Database\Seeds\ClientsSeeder.php(114): CodeIgniter\BaseModel->save(Array)
#9 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\ClientsSeeder->run()
#10 X:\00.templates\ci4-starter\dist\app\Database\Seeds\A2cms.php(13): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#11 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
#12 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#13 X:\00.templates\ci4-starter\dist\app\Commands\Install.php(53): App\Commands\Install->runSeeder()
#14 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Commands.php(70): App\Commands\Install->run(Array)
#15 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', Array)
#16 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#17 X:\00.templates\ci4-starter\dist\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#18 X:\00.templates\ci4-starter\dist\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#19 {main}
CRITICAL - 2024-11-29 17:38:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Table '808_starter.clients_contacts' doesn't exist
[Method: CLI, Route: starter:install]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clients_contacts` (`firstname`, `lastname`, `type`, `mail`, `cellphone`, `workphone`, `created_at`, `updated_at`) VALUES (:firstname:, :lastname:, :type:, :mail:, :cellphone:, :workphone:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Database\Seeds\ClientsSeeder.php(114): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\ClientsSeeder->run()
 8 APPPATH\Database\Seeds\A2cms.php(13): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\ClientsSeeder')
 9 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
10 APPPATH\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\A2cms')
11 APPPATH\Commands\Install.php(53): App\Commands\Install->runSeeder()
12 SYSTEMPATH\CLI\Commands.php(70): App\Commands\Install->run([])
13 SYSTEMPATH\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', [])
14 SYSTEMPATH\Boot.php(351): CodeIgniter\CLI\Console->run()
15 SYSTEMPATH\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
16 ROOTPATH\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
CRITICAL - 2024-11-29 17:38:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table '808_starter.clients_contacts' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clients_contacts` (`firstname`, `lastname`, `type`, `mail`, `cellphone`, `workphone`, `created_at`, `updated_at`) VALUES (\'Jay\', \'Lamping\', \'main\', \'jay@808.biz\', \'808-725-1900\', \'808-600-7400\', \'2024-11-29 17:38:39\', \'2024-11-29 17:38:39\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clients_contacts` (`firstname`, `lastname`, `type`, `mail`, `cellphone`, `workphone`, `created_at`, `updated_at`) VALUES (\'Jay\', \'Lamping\', \'main\', \'jay@808.biz\', \'808-725-1900\', \'808-600-7400\', \'2024-11-29 17:38:39\', \'2024-11-29 17:38:39\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clients_contacts` (`firstname`, `lastname`, `type`, `mail`, `cellphone`, `workphone`, `created_at`, `updated_at`) VALUES (:firstname:, :lastname:, :type:, :mail:, :cellphone:, :workphone:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Database\Seeds\ClientsSeeder.php(114): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\ClientsSeeder->run()
10 APPPATH\Database\Seeds\A2cms.php(13): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\ClientsSeeder')
11 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
12 APPPATH\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\A2cms')
13 APPPATH\Commands\Install.php(53): App\Commands\Install->runSeeder()
14 SYSTEMPATH\CLI\Commands.php(70): App\Commands\Install->run([])
15 SYSTEMPATH\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', [])
16 SYSTEMPATH\Boot.php(351): CodeIgniter\CLI\Console->run()
17 SYSTEMPATH\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
18 ROOTPATH\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
CRITICAL - 2024-11-29 17:38:39 --> [Caused by] mysqli_sql_exception: Table '808_starter.clients_contacts' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `clients_contacts` (`firstname`, `lastname`, `type`, `mail`, `cellphone`, `workphone`, `created_at`, `updated_at`) VALUES (\'Jay\', \'Lamping\', \'main\', \'jay@808.biz\', \'808-725-1900\', \'808-600-7400\', \'2024-11-29 17:38:39\', \'2024-11-29 17:38:39\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clients_contacts` (`firstname`, `lastname`, `type`, `mail`, `cellphone`, `workphone`, `created_at`, `updated_at`) VALUES (\'Jay\', \'Lamping\', \'main\', \'jay@808.biz\', \'808-725-1900\', \'808-600-7400\', \'2024-11-29 17:38:39\', \'2024-11-29 17:38:39\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clients_contacts` (`firstname`, `lastname`, `type`, `mail`, `cellphone`, `workphone`, `created_at`, `updated_at`) VALUES (\'Jay\', \'Lamping\', \'main\', \'jay@808.biz\', \'808-725-1900\', \'808-600-7400\', \'2024-11-29 17:38:39\', \'2024-11-29 17:38:39\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clients_contacts` (`firstname`, `lastname`, `type`, `mail`, `cellphone`, `workphone`, `created_at`, `updated_at`) VALUES (:firstname:, :lastname:, :type:, :mail:, :cellphone:, :workphone:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Database\Seeds\ClientsSeeder.php(114): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\ClientsSeeder->run()
11 APPPATH\Database\Seeds\A2cms.php(13): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\ClientsSeeder')
12 SYSTEMPATH\Database\Seeder.php(147): App\Database\Seeds\A2cms->run()
13 APPPATH\Commands\Install.php(87): CodeIgniter\Database\Seeder->call('App\\Database\\Seeds\\A2cms')
14 APPPATH\Commands\Install.php(53): App\Commands\Install->runSeeder()
15 SYSTEMPATH\CLI\Commands.php(70): App\Commands\Install->run([])
16 SYSTEMPATH\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('starter:install', [])
17 SYSTEMPATH\Boot.php(351): CodeIgniter\CLI\Console->run()
18 SYSTEMPATH\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
19 ROOTPATH\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
